packages/terra-core-docs/src/terra-dev-site/doc/form-input/About.1.doc.mdx

```diff
@@ -26,6 +27,12 @@ Generic input which represents an HTML input element directly.
 
 **Note:** To follow [UX Best Practices](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Placeholder_Research), this component does not allow `placeholder`.
 
+<Notice variant="caution" ariaLevel="3">
+
+**Accessibility Bug:** Due to a bug in Chromium-based browsers (Chrome, Microsoft Edge, etc.), VoiceOver would read `<input type="number" />` incorrectly when incrementing or decrementing the value with keyboard shortcuts. Terra Form Input utilizes `<input type="number" />` when the `type` prop is set to "number" and will also have this screenreader issue. An alternative is to use something like `<input type="text" inputmode="numeric" pattern="\d*" />`. However, this will have no built in validity checks or keyboard commands. Consumers are responsible for adding this functionality.
```

Comment:
Based on [this clarifying comment](https://github.com/cerner/terra-core/pull/3840#discussion_r1258931306 ), I think we should specify that we're using the vanilla html input since w're switching context from the previous sentence. Suggested changes:

````suggestion
**Accessibility Bug:** Due to a bug in Chromium-based browsers (Chrome, Microsoft Edge, etc.), VoiceOver would read `<input type="number" />` incorrectly when incrementing or decrementing the value with keyboard shortcuts. Terra Form Input utilizes `<input type="number" />` when the `type` prop is set to "number" and will also have this screenreader issue. A recommended solution would be to use the vanilla html `input` instead of Terra Form Input. For example: 

 ```html
<input type="text" inputmode="numeric" pattern="\d*" />
```

However, this will not have the built-in validity checks or keyboard commands that Terra Form Input provides. Consumers are responsible for adding this functionality.
````

CC: @eawww for accuracy of the statements.

Reply:
nit/question: Is "vanilla" as a synonym for "ordinary" fairly universal among English speakers from assorted cultural backgrounds? I assumed it was specific to America's specific variety of ice cream obsession.

Reply:
We could use "basic html"  or "html standard" instead of "vanilla html".

Reply:
Yeah `basic html` is fine. I said vanilla because vanilla jsavascript is a relatively common term but basic sounds more clearer.

Reply:
Changed here https://github.com/cerner/terra-core/pull/3840/commits/fbc8fe60c17feb83830edb4adf0f8581b9b84e0f